src/components/projects/new-project.jsx

```diff
@@ -57,8 +69,15 @@ export default function NewProject({ isOpen, onOpen, onClose }) {
             {(props) => (
               <Form>
                 <Stack spacing='2'>
-                  <FormikField name="name" label="Project name" validation={stringIsNotEmpty} />
+                  <Tooltip label>
```

Comment:
what does `label` do here? shouldnt it be set to something?

Reply:
omg I did not mean to push this